CHANGELOG.rst

```diff
@@ -21,6 +21,10 @@ Added
 - Add `no_key` parameter to `queryset.select_for_update`.
 - `F()` supports referencing JSONField attributes, e.g. `F("json_field__custom_field__nested_id")` (#1960)
 
+Fixed
```

Comment:
0.25.1 has been already released, you need to add `0.25.2 (unreleased)` and put your changes there.